cmd/policyrules_cmd.go

```diff
@@ -110,10 +111,18 @@ rbac-tool policy-rules -o json  | jp "[? @.allowedTo[? (verb=='get' || verb=='*'
 				policies := rbac.NewSubjectPermissionsList(filteredPolicies)
 
 				for _, p := range policies {
+
+					var subject string
+					if p.Subject.Kind == "ServiceAccount" {
+						subject = fmt.Sprintf("%v:%v", p.Subject.Namespace, p.Subject.Name)
```

Comment:
I suggest using here '/' instead of ':'

Reply:
I have just done this update. Thanks and sorry for the delay